### ISSUE TITLE
is still recommended?

### ISSUE THREAD

**Issue by djamaile** [andreimerfu/gitlab_roulette#6]
title. and if not what are the alternatives?

**Comment by andreimerfu** [comment]
No, it's deprecated now. Meanwhile, GitLab has created a dedicated gem that does the same thing.

https://gitlab.com/gitlab-org/ruby/gems/gitlab-dangerfiles
